isb_lib/vocabulary/vocab_adapter.py

```diff
@@ -37,11 +38,15 @@ def _read_descendants(term: Term, repository: TermRepository) -> dict:
     return term_dict
 
 
-@functools.lru_cache(maxsize=10)
```

Comment:
After turning on SQL logging, it became apparent that this didn't work as intended.  The `repository` parameter is part of the cache key, and it was preventing things from being cached.  Rather than trying to make `TermRepository` be cacheable, it seemed clearer to just make this work out of a dictionary.